libraries/ESP8266WiFi/src/include/UdpContext.h

```diff
@@ -167,6 +167,26 @@ class UdpContext
 
 #endif // !LWIP_IPV6
 
+    /*
+     * Add a netif (by its index) as the multicast interface
+     */
+    void setMulticastInterface(netif* p_pNetIf)
+    {
+#if LWIP_VERSION_MAJOR == 1
+        udp_set_multicast_netif_addr(_pcb, (p_pNetIf ? ip_2_ip4(p_pNetIf->ip_addr) : ip_addr_any));
```

Comment:
`ip_2_ip4()` is useless with lwIP-1.4
https://github.com/esp8266/Arduino/blob/00440cd84a4ed9467de6305cb768875e09c12303/cores/esp8266/IPAddress.h#L33